src/Tidy/Codegen.purs

```diff
@@ -896,7 +909,7 @@ binderVar = BinderVar <<< toName
 
 -- | Constructs a named binding pattern (`@`).
 binderNamed :: forall e name. ToName name Ident => name -> Binder e -> Binder e
-binderNamed n = BinderNamed (toName n) tokAt
+binderNamed n = BinderNamed (toName n) tokAt <<< binderParens
```

Comment:
This should be `precBinder2` rather than `binderParens`, I think.

Reply:
Ah.. I thought usage of `@` in a binder always required a parens. On the other hand, I don't think `precBinder2` is quite right because a constructor doesn't need a parens. I tried this out on Try PureScript and didn't get an error or warning:
```

foo :: forall a. a -> String
foo = case _ of
 a@b -> ""

foo2 :: forall a. a -> String
foo2 = \a@b -> ""

data Foo = Foo

foo3 :: Foo -> String
foo3 = \a@Foo -> ""